File: src/Leetcode_14.java
class Leetcode_14 {
    public String longestCommonPrefix(String[] strs) {
        for(int i= 1; i < strs.length ; i++){
            while(strs[i].indexOf(strs[0]) != 0){
                strs[0]= strs[0].substring(0, strs[0].length() - 1);
            } 
        }
        return strs[0];
    }
}

File: python/Leetcode_14.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        if not strs: 
            return ""
        for i in range (1, len(strs)):
            while strs[i].find(strs[0]) != 0:
                strs[0]= strs[0][:-1]  
                if not strs[0]: 
                    return ""
        return strs[0]